sdks/python/apache_beam/transforms/stats.py

```diff
@@ -192,7 +193,7 @@ def get_estimate(self):
     if len(self._sample_heap) < self._sample_size:
       return len(self._sample_heap)
     else:
-      sample_space_size = sys.maxsize - 1.0 * self._min_hash
+      sample_space_size = self._HASH_SPACE_SIZE - 1.0 * self._min_hash
```

Comment:
Ah, that was because `python hash()` can return negative values. So yes, I think we need to set `self._min_hash = 2**64` given that now all hash sizes are positive.

Reply:
Valentyn-- you're right, forgot to update self._HASH_SPACE_SIZE. I think originally pythons hash() can return negative value, so the total hash space ranges from (- sys.maxsize,  sys.maxsize).

Reply:
Modified it to 2^63 - 1 (which is also equivalent to sys.maxsize... which is why my tests were passing.)